testing/features/support/components/search/standard.rb

```diff
@@ -18,6 +18,7 @@ def search_for(term)
     private
 
     def clear_field
+      search_field.send_keys(%i[control end]) if firefox?
```

Comment:
Is this something you were debugging? This shouldn't be required on firefox>?

Reply:
The tests were falling for Firefox, then I realized that just send_keys(:end) was not sending the caret navigation to the end of the text, resulting in the following error:

<img width="861" alt="Screenshot 2021-02-26 at 13 14 57" src="https://user-images.githubusercontent.com/11262774/109304804-afeb5480-7834-11eb-99d4-ed7f4df7c2c6.png">

```
RSpec::Expectations::ExpectationNotMetError: 
expected: "Anything"
     got: "AnythingCurrent search term" 
```

Reply:
Right, oki. This shouldn't be the case. See if you can find out what's causing it. I'll dig into it on Monday too if needs be.

This should work for all browsers. If it's not something is likely wrong with our code.

Reply:
In the end, I didn't find out why `search_field.send_keys(:end)` wasn't working properly on Firefox.
I just find out that the array of symbols format works fine for Firefox `search_field.send_keys(%i[end])`, unlike the default `search_field.send_keys(:end)`. The array of symbol works fine on chrome too.